pom.xml

```diff
@@ -74,10 +83,15 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>script-security</artifactId>
-            <version>1.56</version>
             <scope>test</scope>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.main</groupId>
+            <artifactId>jenkins-test-harness-htmlunit</artifactId>
+            <version>154.v973ee207a_b_95</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Why is this being added?

Reply:
I can't recall the exact reason why I had to add it once to make it work, but it's no longer necessary. Let's remove it.